test/fixtures/version-4/js-version-not-in-url-4.0.0.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>Test</title>
+        <!--[if lt IE 9]>
+            <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+            <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+        <![endif]-->
+        <script src="../../lib/jquery.min.js"></script>
+        <script>
+        (function () {
+            var modal = window.$.fn.modal = {};
```

Comment:
@hnrch02 I've converted this test and the two tests which do this in `fixtures/outdated` to use this style mock instead of the full constructor as was done before.

It needs to be `window.$.fn.modal = ...` though to pass the tests :+1: